src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/01 instrumentHTTP.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'instrumentHTTP'
+excerpt: 'instrumentHTTP instruments the k6 http module with tracing capabilities.'
+---
+
+The `instrumentHTTP` function instruments the k6 http module with tracing capabilities. It will transparently replace each of the k6 http module [del](/javascript-api/k6-http/del), [get](/javascript-api/k6-http/get), [head](/javascript-api/k6-http/head), [options](/javascript-api/k6-http/options), [patch](/javascript-api/k6-http/patch), [post](/javascript-api/k6-http/post), [put](/javascript-api/k6-http/head), and [request](/javascript-api/k6-http/request) functions with versions that will automatically attach a trace context to every request.
```

Comment:
```suggestion
The `instrumentHTTP` function instruments the k6 http module with tracing capabilities. It transparently replaces each of the k6 http module functions with versions that automatically attach a trace context to every request. Functions that you can instrument include [del](/javascript-api/k6-http/del), [get](/javascript-api/k6-http/get), [head](/javascript-api/k6-http/head), [options](/javascript-api/k6-http/options), [patch](/javascript-api/k6-http/patch), [post](/javascript-api/k6-http/post), [put](/javascript-api/k6-http/head), and [request](/javascript-api/k6-http/request).
```

Reply:
> transparently

I'd recommend taking this out, since it seems like a subjective term. If it's important to explain what transparent means, I'd add an explanatory sentence in the middle:

> The `instrumentHTTP` function instruments the k6 http module with tracing capabilities.
> It replaces each of the k6 http module functions with versions that automatically attach a trace context to every request.
> **The process is transparent because <explanation>....**
> Functions that you can instrument include [


This seems unnecessary, but I don't know what information "transparently" suggests here. Your call.

Reply:
I think you're current suggestion is correct. I wouldn't want to explain transparent here as it doesn't have any specific technical meaning: it stand for "without further interactions from the user" 👍🏻